src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -756,13 +753,19 @@ export const GlobalProvider = ({
                             parentNode: parentId,
                             selected: false,
                         };
+                        delete returnData.handleBounds;
+                        return returnData;
                     });
                 return [...nodes, ...newNodes];
             });
 
             changeEdges((edges) => {
                 const newEdges = edges
-                    .filter((e) => nodesToCopy.has(e.target))
+                    .filter(
+                        (e) =>
+                            (nodesToCopy.has(e.target) && nodesToCopy.has(e.source)) ||
```

Comment:
Why did you change how edges are copied?

Reply:
The target and source need to exist for the edge to work, and I only want to copy it if it was copied. I now realize this condition is wrong for that though